internal/common/resources.go

```diff
@@ -0,0 +1,29 @@
+package common
+
+import (
+	"os"
+	"path/filepath"
+)
+
+const (
+	API_DESCRIPTION_FILENAME = "api.json"
+	RESOURCES_OUTPUT_DIRNAME = "resources"
+)
+
+func GetResourcesDir() (string, error) {
+	cwd, err := os.Getwd()
+	if err != nil {
+		return "", err
+	}
+
+	return filepath.Join(cwd, "../../", RESOURCES_OUTPUT_DIRNAME), nil
```

Comment:
!
Careful, subtle bug here.

You can't use "../.." in a path join, since you already joined two things in that string.
You need to join each "..".
For example:
https://github.com/edulinq/autograder-server/blob/main/internal/util/version.go#L53